src/Microsoft.DotNet.Wpf/src/System.Xaml/System/Windows/Markup/NameReferenceConverter.cs

```diff
@@ -27,7 +24,7 @@ public override object ConvertFrom(ITypeDescriptorContext context, CultureInfo c
         {
             if (context == null)
             {
-                throw new ArgumentNullException("context");
+                throw new ArgumentNullException(nameof(context));
```

Comment:
This will conflict with PR #173 https://github.com/dotnet/wpf/pull/173/files#diff-50e59c71cb48d2462ce7e06b2cf17a6d